src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,62 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Nested
+    @DisplayName("요구사항 1")
+    class Requirement1 {
+
+        @Test
+        @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인하는 학습 테스트")
+        void size() {
+            assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
assertThat(numbers).hasSize(3);
```

이런식으로도 작성이 가능합니다. 좀 더 직관적일수도 있을거 같아요. assertJ는 여러가지 메서드들을 제공해줍니다.

Reply:
더 직관적이고 휴머너블 하네요.
다음에 반영해봐야겠습니다!